src/map/status.h

```diff
@@ -2074,14 +2074,15 @@ struct status_data {
 		hp, sp,  // see status_cpy before adding members before hp and sp
 		max_hp, max_sp;
 	short
-		str, agi, vit, int_, dex, luk;
-	unsigned short
-		batk,
+		str, agi, vit, int_, dex, luk,
+		batk;
```

Comment:
@aleos89 I tried it and as expected it works fine in RE. but when in PRE it doesn't work well. The computation for each mode is different in approach.